src/Evolution/Systems/CurvedScalarWave/BoundaryCorrections/BoundaryCorrection.hpp

```diff
@@ -0,0 +1,44 @@
+// Distributed under the MIT License.
```

Comment:
if a system has a non-flat Cartesian metric then the `inverse_spatial_metric` tag must be specified in the system, otherwise you'll get incorrect normal vectors

Reply:
Just added the `inverse_spatial_metric_tag` to System, thanks!